queries.sql

```diff
@@ -8,4 +8,70 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+
+UPDATE animals SET species = 'Unspecified';
+
+SELECT * FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+BEGIN ;
+
+UPDATE animals SET species = 'Digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species = 'Pokemon' WHERE species IS NULL;
+
+SELECT * FROM animals;
+
+COMMIT;
+
+SELECT * FROM animals;
+
+BEGIN ;
+
+DELETE FROM animals;
+
+SELECT * FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+BEGIN ;
+
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT my_savepoint;
+
+UPDATE animals SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO SAVEPOINT my_savepoint;
+
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+
+COMMIT;
+
+SELECT COUNT(*) AS total_animals FROM animals;
+SELECT COUNT(*) AS animals_never_escaped FROM animals WHERE escape_attempts = 0;
```

Comment:
- [ ] Great job ensuring that your screenshots have the queries they belong to as the previous reviewer mentioned:wink: 
- [ ] I noticed that the results for these 2 queries aren't returning the right results. If you followed the previous project instructions and this one properly, then the total number of animals that should have returned is 11 from the beginning and after deleting one in the later queries, then it should be left with 10. But you have had 10 animals from the start of all the queries.
- [ ] I also noticed from your query results that Angemon returned twice. This means you're missing 2 animals instead of 1 so kindly fix that as well :+1: 
- Screenshot of your Query Results
![image](https://github.com/waelelsafty07/vet/assets/102699958/bc64f088-4f7f-4afd-846f-0630bb80fa3f)

- [ ] Also, the escape attempt query should also return 2 instead of 1 so kindly fix those :100: